### PR TITLE
Add icon 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the default React icon is used for our favicon. #6 

**What is the new behavior (if this is a feature change)?**
- A new icon has been designed and implemented. #6

### CODE REVIEW

File: public/fav/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig><msapplication><tile><square70x70logo src="/ms-icon-70x70.png"/><square150x150logo src="/ms-icon-150x150.png"/><square310x310logo src="/ms-icon-310x310.png"/><TileColor>#ffffff</TileColor></tile></msapplication></browserconfig>
```
Comment (kleberbaum) [inline public/fav/browserconfig.xml]: It would be much better with some line breaks.

**Review by kleberbaum — CHANGES_REQUESTED** [review]
After this it should be ok to merge.

File: public/fav/browserconfig.xml (hunk 1 above)
Comment (Aichnerc) [reply public/fav/browserconfig.xml]: The formatting of this document has been changed with my latest commit.

**Review by Aichnerc — COMMENTED** [review]
Everything seems clean now.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**